unit/ctest_polarization.c

```diff
@@ -0,0 +1,310 @@
+#include "gkyl_array.h"
```

Comment:
@Maxwell-Rosen please adapt this unit test so it also runs on GPUs. See other unit tests, for example ctest_positivity_shift. With one variation, instead of using this
```
  distf_ho = distf;
  if (use_gpu)
    distf_ho = mkarr(false, distf->ncomp, distf->size);
...
  if (use_gpu)
    gkyl_array_release(distf_ho);
```
It is best to use this structure:
```
  distf_ho = use_gpu? mkarr(false, distf->ncomp, distf->size)
                    : gkyl_array_acquire(distf);
...
  gkyl_array_release(distf_ho);
```